example/package.json

```diff
@@ -21,6 +22,7 @@
     "react": "18.2.0",
     "react-native": "0.73.4",
     "react-native-reanimated": "^3.5.4",
+    "react-native-gesture-handler": "*",
```

Comment:
please specify a version, don't use "*" - it may break something for someone when there is a new release of GH